make/photon/chartserver/Dockerfile

```diff
@@ -2,21 +2,26 @@ FROM photon:2.0
 
 RUN tdnf install -y shadow sudo >>/dev/null\
     && tdnf clean all \
-    && mkdir /chartserver/ \
-    && mkdir /harbor/ \
-    && groupadd -r -g 10000 chartuser \
-    && useradd --no-log-init -m -r -g 10000 -u 10000 chartuser
-COPY ./make/photon/chartserver/binary/chartm /chartserver/
-COPY ./make/photon/chartserver/docker-entrypoint.sh /docker-entrypoint.sh
-COPY ./make/photon/common/install_cert.sh /harbor
+    && groupadd -r -g 10000 chart \
+    && useradd --no-log-init -m -g 10000 -u 10000 chart
+
+COPY ./make/photon/chartserver/binary/chartm /home/chart/
+COPY ./make/photon/chartserver/docker-entrypoint.sh /home/chart/
+COPY ./make/photon/common/install_cert.sh /home/chart/
+
+RUN chmod -R 777 /etc/pki/tls/certs \
```

Comment:
Is chmod 777 really the security conscious thing to be doing to the sensitive certs directory?

Reply:
777 is not secure, so change the owner to another user